Mlem/App/Views/Pages/Community/CommunityView.swift

```diff
@@ -114,9 +114,16 @@ struct CommunityView: View {
             VStack {
                 switch selectedTab {
                 case .posts:
-                    if let postFeedLoader {
-                        postsTab(community: community, postFeedLoader: postFeedLoader)
-                            .padding(.bottom, -4)
+                    VStack {
```

Comment:
*nit*
```suggestion
                    Group {
```

Unless there's some sneaky `VStack` behavior being used here

Reply:
`Group` has weird behavior with modifiers. The group itself can't have modifiers, so when you apply a modifier it applies it individually to each child view. This causes the modifier to only apply when the `postFeedLoader` is non-nil, so we need to use a "real" container view here.

Reply:
I agree that `VStack` seems weird. Maybe `ZStack` is better? Or `AnyView` maybe. Or we could define our own bare-bones container view for this purpose.

Reply:
I agree that `VStack` seems weird. Maybe `ZStack` or `AnyView` would be better? Or we could define our own bare-bones container view for this purpose.

Reply:
Gotcha. `VStack`, `ZStack`, and `AnyView` are basically equally janky IMO. A container would be neatest, but I'm fine merging with `VStack`.

Reply:
I just tried defining a custom `SimpleContainer` type, but it produces the same output as the `Group` 🤔 Will stick with `VStack` for now.